spec/unit/create.spec.js

```diff
@@ -300,4 +301,23 @@ describe('create', function () {
             });
         });
     });
+
+    describe('writeNameForAndroidStudio', () => {
+        var project_path = path.join('some', 'path');
```

Comment:
In general we want to avoid using `var`, and instead we use `let` or `const` depending on if we expect the variable to change.

```suggestion
        const project_path = path.join('some', 'path');
```

Reply:
Well, I love ES6 (and TypeScripe) but when I always try to follow the pattern of existing code and found that existing code is using `var` mostly and using snake case variables, so used `var`. Thanks anyways. Updated this for new code.

Reply:
> Well, I love ES6 (and TypeScripe) but when I always try to follow the pattern of existing code and found that existing code is using var mostly and using snake case variables, so used var. Thanks anyways. Updated this for new code.

Yah, `var` refactoring is a work in progress which I think is why we don't have an eslint config (?) I was kinda surprised the eslint didn't produce an error on it tbh.